### ISSUE TITLE
Can we make this demo look awesome!

### ISSUE THREAD

**Issue by skifaster** [meteor-seattle/meteor-101-codefellows#1]
Anything you have in mind would be fine. Let's do this in a branch or tag, so we can pull and deploy at the very end of the coding period.

**Comment by skifaster** [comment]
@warmbowski suggested using  S-alert http://s-alert-demo.meteor.com/, for the winner announcement ...

**Comment by bryanterrell** [comment]
Yeah, I like the S-alert - pretty slick. I will take a look at it tonight.
